SocialCareCaseViewerApi/V1/Gateways/DatabaseGateway.cs

```diff
@@ -234,9 +236,12 @@ public string GetNCReferenceByPersonId(string personId)
             return lookup?.NCId;
         }
 
-        public List<Worker> GetWorkers(int teamId)
+        public List<Worker> GetWorkers(int? teamId = null, int? id = null)
```

Comment:
Please review the input parameter setup. Due to way the framework works, these values will never be null. They can be int default values though (0), so please update the parameters to reflect that